core/tests/protractor_utils/StoryEditorPage.js

```diff
@@ -101,11 +102,35 @@ var StoryEditorPage = function() {
   var warningIndicator = element(by.css('.protractor-test-warning-indicator'));
   var warningTextElements = element.all(
     by.css('.protractor-test-warnings-text'));
+  var storyThumbnailImageElement = element(
+    by.css('.story-thumbnail .protractor-test-custom-photo'));
+  var storyThumbnailClickable = element(
```

Comment:
This name actually sounds more like a boolean ("story thumbnail is clickable"). Maybe rename it to something else that sounds more like a noun?

Ditto for chapterThumbnailClickable, etc.

Reply:
Done.